### ISSUE TITLE
Feature request: User tags

### ISSUE THREAD

**Issue by danielgindi** [cuckoosandbox/cuckoo#958]
Is there an option to add tags to a new task - but not for machine selection?

Basically a user should be able to add a "category" or a some kind of context. It could be "debug"/"test"/"phase2" tags etc.

**Comment by jbremer** [comment]
You can fill the `custom` field as desired. Does that suffice or would you like to actually use the given value for some other analysis criteria in Cuckoo? (The `custom` field is left as-is and doesn't change the analysis in any way, aside from passing along a value into the final report).

**Comment by danielgindi** [comment]
What I mean is actual tags, so you can "split" the database based on a tag or even multiple tags defined for a report. So I can safely say that the "custom" field does not suffice :-)

I'm looking into how to implement this in Cuckoo without shaking the ground

**Comment by botherder** [comment]
Yep, that's been on a todo list for a while. I thought there was already a ticket open for it, but apparently not.

**Comment by jbremer** [comment]
Any update on your efforts in this matter @danielgindi?

**Comment by danielgindi** [comment]
As I was digging in I found out that I need to be more familiar with the different aspects - the mongodb, the local sqlite, the apis, and more and more - just to add those tags... So I just dealt with it.
I usually implement stuff that I need, but in this case I just didn't have the time...

**Comment by jbremer** [comment]
Fair enough. I don't have any ETA, but we'll be taking a look at this request sometime :)
First finishing off some bigger features though.
